tests/README.md

```diff
@@ -0,0 +1,75 @@
+### Integrated tools for synthetic data generation
+
+PyTrousse aids developers in automated testing by creating simulated raw data in a controlled way along with the data expected after appropriate data wrangling operations.
+Particularly this tool can support the data transformation operator testing by inverting their behavior through error injection.
+
+Particularly raw and expected data are wrapped in a `TestDataset` object that, following PyTrousse behavior, can be modified by applying the inverted data transformation operators.
+```python
+>>> import pandas as pd
+from .datasim import from_pandas
+from .datasim_util import (
+    Compose,
+    InsertOutOfScaleValues,
+    ReplaceSubstringsByValue,
+    SubstringReplaceMapByValue,
+)
+
+>>> df = pd.DataFrame(
+    {"float_column": list(range(10)), "int_column": [0.2 * i for i in range(10)]}
+)
+test_dataset = from_pandas(df)
+insert_errors = Compose(
+    [
+        ReplaceSubstringsByValue(
+            column_names=["float_column"],
+            error_count=2,
+            replacement_map_list=[
+                SubstringReplaceMapByValue(
+                    substr_to_replace=".",
+                    new_substring=",",
+                    value_after_fix=".",
+                    fix_with_substring=True,
+                )
+            ],
+        ),
+        InsertOutOfScaleValues(
+            column_names=["int_column"],
+            error_count=2,
+            upperbound_increase=0.02,
+            lowerbound_increase=0.02,
+        ),
+    ]
+)
+test_dataset = insert_errors(test_dataset)
```

Comment:
please add >>> before each python statement and indent accordingly

Reply:
We did not do that in the other README... should I update it too?  I was thinking about '... ' for the rows that are not the first one like in the console when I write python code. What do you think?

Reply:
Mmmmh in the other README we had them indented... but I like the idea of having ....

Reply:
Now it should be better